secboot/encrypt_sb.go

```diff
@@ -38,8 +38,7 @@ import (
 )
 
 var (
-	sbInitializeLUKS2Container       = sb.InitializeLUKS2Container
-	sbAddRecoveryKeyToLUKS2Container = sb.AddRecoveryKeyToLUKS2Container
```

Comment:
Did nothing call this before?

Reply:
It was removed in 3988a93de8b8b97832e9f75cec82d0966ff650a3 which was in PR #11715

This mini key manager has a copy of the internal code of secboot.

I could make a separate commit of removing this and the mock.